utils/wp-odm_related-options.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+  class Wp_odm_related_Options {
+
```

Comment:
It is the same as in **wpckan** where we needed this for writing some of the most important tests.